src/openforms/formio/components/vanilla.py

```diff
@@ -89,6 +95,14 @@ def rewrite_for_request(component: FileComponent, request: Request):
         if component.get("useConfigFiletypes", False):
             config = GlobalConfiguration.get_solo()
             component["filePattern"] = ",".join(config.form_upload_default_file_types)
+            component["file"].update(
+                {
+                    "allowedTypesLabels": [
+                        dict(UploadFileType.choices)[mimetype]
```

Comment:
```suggestion
                        getattr(UploadFileType.choices, mimetype)
```
getattr boils down to a single dict lookup in the `choices.__dict__`, instead of creating a new dict and then doing the lookup on each iteration.

Reply:
`UploadFileType.choices` is actually a list of tuples `(key, value)`, so i'm not sure if there is a way to get around transforming it to a dict :thinking: